### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ecarlosdev**
## Description

Removes references to Lists and things that are not specific to the package.
Add AnimatedBookWidget.builder constructor

## Checklist

- [ ] Tested on iOS
- [ ] Tested on Android
- [ ] Tested on small screens
- [ ] Unit tests

### CODE REVIEW

**Review by ricardogottheil — APPROVED** [review]

**Review by br-programmer — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,156 +9,71 @@ A Flutter package allows you to effortlessly create a animated book widget UI wi
...
 ```dart
 dependencies:
     animated_book_widget: <latest_version>
 ```
+Import:
 
-2.  Import the package
 ```dart
-import 'package:animated_book_widget/animated_book_widget.dart';
+import 'package:animated_book_widget/animated_book_widget.dart
```
Comment (victormanuelfrancodev) [inline README.md:24]: Sorry, why isn't there a ' at the end

File: README.md [hunk 2]
```diff
@@ -9,156 +9,71 @@ A Flutter package allows you to effortlessly create a animated book widget UI wi
...
+  padding: const EdgeInsets.only(right: 10),
+  blurRadius: 8,
+  spreadRadius: .1,
+  backgroundBlurOffset: Offset.zero,
+  backgroundColor: color.withOpacity(.5),
+  backgroundShadowColor: color.withOpacity(.075),
+  curve: Curves.linear,
+  animationDuration: Duration(milliseconds: 500),
+  reverseAnimationDuration: Duration(milliseconds: 500)
+),
```
Comment (victormanuelfrancodev) [inline README.md:53]: when is , or ; ? 

pick one 
 
```
),
```

```
);
```

```
) 
```

Comment (victormanuelfrancodev) [reply README.md:53]: Explain the reason for the modifications and also indicate which ones were made and why. This should be included in the PR description.

**Review by victormanuelfrancodev — COMMENTED** [review]
Explain the reason for the modifications and also indicate which ones were made and why. This should be included in the PR description.

**Pull request merged**